package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "@actions/core": "^1.6.0",
     "chalk": "^4.1.2",
+    "change-case": "^4.1.2",
```

Comment:
Can you include the `package-lock.json` changes for this? The test suite is failing right now because it's missing.